src/daemon/rpmostreed-deployment-utils.cxx

```diff
@@ -539,6 +623,35 @@ rpm_diff_variant_new (RpmDiff *diff)
   return g_variant_dict_end (&dict);
 }
 
+static GVariant *
+manifest_diff_variant_new (ManifestDiff *diff)
```

Comment:
I think it'd work and be a lot simpler to have `GVariant *manifest_diff_variant_from_exported (const rpmostreecxx::ExportedManifestDiff *manifest_diff)` right?

I don't see why we need to create a 3rd intermediate structure.